src/main/java/com/epam/parso/SasFileReader.java

```diff
@@ -54,4 +54,11 @@
      * @return the object of the {@link SasFileProperties} class that stores file metadata.
      */
     SasFileProperties getSasFileProperties();
+    
+    /**
+     * The function to return the index of the current row when reading the file sas7bdat file.
```

Comment:
Please remove redundant 'file' word here and below.

Reply:
this is corrected, thank you.